src/dos/dos_mscdex.cpp

```diff
@@ -253,18 +253,11 @@ int CMscdex::AddDrive(Bit16u _drive, char* physicalPath, Bit8u& subUnit)
 	int result = 0;
 	// Get Mounttype and init needed cdrom interface
 	switch (CDROM_GetMountType(physicalPath,forceCD)) {
-	case 0x00: {	
+	case 0x00:
 		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: Mounting physical cdrom: %s"	,physicalPath);
```

Comment:
Reword `physical cdrom` (only can mount directories and CUEs now)

Reply:
This case (0x00) is still responsible for physical CD-ROMs; in later commit I introduced an enum to make it clear, but I think I can safely remove enum value and this case handling now. I will do it in a new commit.